ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml

```diff
@@ -86,7 +85,7 @@ Item {
             name: "everyone",
```

Comment:
I guess it shouldn't be translated, right? 🤔

Reply:
:|

Reply:
nah, it shouldn't imho